docker/tpm-reset

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+cd /tmp
+
+## kill TPM simulator
+/usr/bin/pkill tpm_server
+
+## remove state file of TPM simulator
+/bin/rm -f NVChip
+
+## start TPM simulator in background
+(/usr/local/bin/tpm_server > /dev/null &)
+
+## wait for TPM simulator to start
+while ! pidof tpm_server >> /dev/null ; do
+    sleep 1
+done
+## additional sleep to compensate for the starting time of the TPM simulator
+sleep 1 
+
+## execute TPM startup
+/usr/bin/env TPM2TOOLS_TCTI_NAME=socket \
+    TPM2TOOLS_SOCKET_ADDRESS=127.0.0.1 \
+    TPM2TOOLS_SOCKET_PORT=2321 \
+    /usr/local/bin/tpm2_startup --clear
```

Comment:
This shouldn't be necessary because `tmp_manufacture.sh -s` does it. Or maybe better leave it here and remove that `-s` so TPM will be ready to use without manufacturing EK cert, that way it will be safe(r) to use on real TPM.

Reply:
When `-s` is not used, the `run-tmux` does not work properly:

```
WARNING:esys:src/tss2-esys/api/Esys_NV_ReadPublic.c:309:Esys_NV_ReadPublic_Finish() Received TPM Error
ERROR:esys:src/tss2-esys/esys_tr.c:210:Esys_TR_FromTPMPublic_Finish() Error NV_ReadPublic ErrorCode (0x0000018b)
ERROR:esys:src/tss2-esys/esys_tr.c:321:Esys_TR_FromTPMPublic() Error TR FromTPMPublic ErrorCode (0x0000018b)
ERROR:esys:src/tss2-esys/esys_iutil.c:1095:esys_GetResourceObject() Error: Esys handle does not exist (70018).
ERROR:esys:src/tss2-esys/api/Esys_NV_ReadPublic.c:176:Esys_NV_ReadPublic_Async() nvIndex unknown. ErrorCode (0x00070018)
ERROR:esys:src/tss2-esys/api/Esys_NV_ReadPublic.c:86:Esys_NV_ReadPublic() Error in async function ErrorCode (0x00070018)
Error: Esys_NV_ReadPublic() esapi:The ESYS_TR resource object is bad
EK certificate can't be read, is this simulated TPM?
```

I guess we can leave it like this for now. We need to add suport for physical TPM anyway somehow later to this environment.